### DISCUSSION TITLE
Cask "LibreWolf": File always corrupt after installation

### DISCUSSION THREAD

**Discussion by fabianschulz** [Homebrew/discussions#5423] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: 9fdb0d53990d3e62e25325a6f89dbb769e84a530
Last commit: 6 days ago
Core tap JSON: 02 Jun 09:49 UTC
Core cask tap JSON: 02 Jun 10:24 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.1 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.1/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hi, 

not sure if this is related to this specifc cask, but I am having difficulties installing LibreWolf as it seems broken after installation. 

The install itself looked as usual:

fabian@MacBook ~ % brew install --cask librewolf
==> Auto-updating Homebrew...
Adjust how often this is run with HOMEBREW_AUTO_UPDATE_SECS or disable with
HOMEBREW_NO_AUTO_UPDATE. Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
==> New Casks
(...)
You have 1 outdated formula installed.
==> Downloading https://raw.githubusercontent.com/Homebrew/homebrew-cask/b9e8f78351f2da3c2cdaae578bcd9b9244246ab5/Casks/l/librewolf.rb
### 100.0%
==> Downloading https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/126.0.1-1/librewolf-126.0.1-1-macos-arm64-package.dmg
### 100.0%
==> Installing Cask librewolf
==> Moving App LibreWolf.app to /Applications/LibreWolf.app
==> Linking Binary librewolf.wrapper.sh to /opt/homebrew/bin/librewolf
🍺  librewolf was successfully installed!

After that when I tried to launch LibreWolf, I am getting this error that the file is apparently damaged:

<img width="249" alt="SCR-20240602-lfji" src="https://github.com/Homebrew/discussions/assets/6419883/07d42c7a-64ac-4f47-b008-be8a55c2e98c">

I then clicked "move to trash" and manually uninstalled it:

fabian@MacBook ~ % brew uninstall librewolf
==> Uninstalling Cask librewolf
==> Unlinking Binary '/opt/homebrew/bin/librewolf'
==> Purging files for version 126.0.1,1 of Cask librewolf
==> Autoremoving 13 unneeded formulae:
(...)

I then made another attempt of installing the LinbrewWolf cask:

fabian@MacBook ~ % brew install --cask librewolf
==> Downloading https://raw.githubusercontent.com/Homebrew/homebrew-cask/b9e8f78351f2da3c2cdaae578bcd9b9244246ab5/Casks/l/librewolf.rb
Already downloaded: /Users/fabian/Library/Caches/Homebrew/downloads/c40fee942a60371599bf408b06e2e93ec94e5b63b7d6fe9ac0da7c207a554f3b--librewolf.rb
==> Downloading https://gitlab.com/api/v4/projects/44042130/packages/generic/librewolf/126.0.1-1/librewolf-126.0.1-1-macos-arm64-package.dmg
Already downloaded: /Users/fabian/Library/Caches/Homebrew/downloads/3ebfb334e225a6e2c770145bf9948764562806e4ba29008be57d50dc9f5ea95f--librewolf-126.0.1-1-macos-arm64-package.dmg
==> Installing Cask librewolf
==> Moving App LibreWolf.app to /Applications/LibreWolf.app
==> Linking Binary librewolf.wrapper.sh to /opt/homebrew/bin/librewolf
🍺  librewolf was successfully installed!

But even after that, I get the same error as seen in the screenshot.

Any ideas how I can proceed?

### Relevant casks

LibreWolf

**Comment by Bo98** [comment, marked as answer]
This is an upstream issue: https://gitlab.com/librewolf-community/browser/bsys6/-/issues/14 (with a workaround)

**Comment by fabianschulz** [comment]
Thank you @Bo98 I was not aware of that - the workaround in the GitLab ticket "fixed" it. Appreciate your help!
